source/core/slang-shader-cache-index.h

```diff
@@ -0,0 +1,74 @@
+// slang-shader-cache-index.h
```

Comment:
Why is this in core? It should be in `gfx` for now. `core` is for basic C++ classes like `List`, `RefPtr` etc. and not for any actual compiler/gfx logic.